ciao-cli/image.go

```diff
@@ -97,18 +87,11 @@ var imageCommand = &command{
 }
 
 type imageAddCommand struct {
-	Flag            flag.FlagSet
-	name            string
-	id              string
-	containerFormat imageContainerFormat
-	diskFormat      imageDiskFormat
-	minDiskSize     int
```

Comment:
When workloads CRUD capabilities are available, it is really useful to have Disk and Memory minimal values for Workloads launching.